LoopKitUI/Views/ResizeablePicker.swift

```diff
@@ -9,15 +9,15 @@
 import SwiftUI
 import UIKit
 
-public struct ResizeablePicker<SelectionValue>: UIViewRepresentable where SelectionValue: CustomStringConvertible & Hashable {
+struct ResizeablePicker<SelectionValue>: UIViewRepresentable where SelectionValue: CustomStringConvertible & Hashable {
```

Comment:
I made this internal to allow `QuantityPicker` and `FractionalQuantityPicker` to work as they did since they depend on this. I didn't see the need to update these pickers because the complexity of the configuration options makes me believe UIPicker is still the right option here.